kythe/cxx/indexer/cxx/testdata/basic/namespace_vnames.cc

```diff
@@ -5,17 +5,16 @@
 //- @namespace ref vname(_,"bundle","","test.cc","c++")
 namespace { }
 
-// Named namespaces drop path and root but keep corpus.
-//- @ns ref vname(_,"bundle","","","c++")
+// Named namespaces drop path and root and use the default corpus.
```

Comment:
Is this test any weaker because the default corpus is "" ? Can/should we make a distinct non-empty default corpus in play for the tests?

Reply:
I don't believe so, no.  There is a specific test that the default corpus behaves as expected.